ros2param/test/test_cli_delete.py

```diff
@@ -0,0 +1,188 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
```

Comment:
@BMarchi I presume you needed a separate separate file because otherwise tests would affect each other. Consider doing so for the `set` command as well, but instead of duplicating the entire test infrastructure you could split it out in a separate module and reuse.